roles/benchmarking_deploy_coco_dataset/tasks/main.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Ensure that a hostname was passed in parameter
+  fail: msg="'benchmarking_node_hostname' must be provided"
+  when: benchmarking_node_hostname | default('', true) | length == 0
+
+- name: Ensure that a valid hostname name was passed in parameter
+  command: oc get nodes -l kubernetes.io/hostname={{ benchmarking_node_hostname }} -oname
+
+- name: Delete the mirror secret, if it exists
```

Comment:
Consider a more idempotent approach, or protect this command with an `--allow-overwrite` toolbox flag

Reply:
Oh sorry, you're just deleting the secret, not the mirror. Ignore me

Reply:
hum, you though it was the PVC that was being deleted I guess ;-)
no, no command doing that for now ;-)

the test is actually validating that the command is idempotent, and doesn't  download twice the dataset (and no more md5sum during the 2nd try, only a simple flag-file check ;-))
```
for i in 1 2; do
    ./run_toolbox.py benchmarking download_coco_dataset \
                     "$gpu_node_hostname" \
                     --mirror_base_url=https://mirror-dataset.apps.ci-mirror.psap.aws.rhperfscale.org/coco \
                     --client-cert=${PSAP_SECRET_PATH}/entitled-mirror-client-creds.pem
done
```